src/index.html

```diff
@@ -8,6 +8,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+      </div>
+      <div class="stars stars--1">
+      </div>
```

Comment:
Each of this blocks must have 5 children blocks with stars, and you should
```suggestion
      <div class="stars stars--1">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```
And according to `--1` modifier you should render for each star either `star.svg` or `star-active.svg` as a background using `:nth-child`

stars stars--1 means that you need to render 1 active star
stars stars--3 - 3 active stars
and so on